### ISSUE TITLE
How do you populate `config.robot.admins` for this handler?

### ISSUE THREAD

**Issue by jayroh** [kenjij/lita-slack#12]
I'm trying to assign admins for our slack bot but I can't figure out what's required in that admins array. Is it the user's name? Their slack id? After some digging I did find this tool at slack to retrieve that info but I'm having a hard time getting the group add/delete functionality to work - https://api.slack.com/methods/users.list/test

I've added our user id's to the config, restarted the bot, and when I run `<bot name> auth add <my name> test_group` it does nothing.

Any pointers?

**Comment by jayroh** [comment]
Nevermind - I think I got this working, but there's a lack of documentation around this. 

I'll close this and see about helping clarify it in a PR for the readme.

**Comment by jimmycuadra** [comment]
Thanks for reporting this - you are right that it's not only a big gap in the documentation but also a big usability problem, since there is no easy way to get the Slack IDs that Lita wants for that configuration attribute. The issue has actually been addressed in jimmycuadra/lita#72, and will be included in the next minor release of Lita, at which point the documentation here can reference that new built-in handler as a way of determining what values to use when configuring the Slack adapter.

**Comment by Gaurav2Github** [comment]
I am also trying to load the admins for my bot which is using hipchat adapter.
Adding user id in my lita config, provided by following 'users find' command 
```
## in hipchat
Lita users find <username>
<username> (ID: <userid>, Mention name: <mentionname>)

## in lita_config.rb
config.robot.admins = ["userid"]
````
After loading this config. It does not load any auth users
```
## in hipchat
Lita auth list
There are no authorization groups yet
```
Any suggestions, what can be done to load admins or admin groups?
